sdks/go/pkg/beam/io/healthcare/fhirio/client.go

```diff
@@ -0,0 +1,39 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specifinguage governing permissions and
+// limitations under the License.
+
+package fhirio
```

Comment:
Should this be in the same package? I guess there isn't necessarily a reason to split them up, either is fine.

Reply:
I think [this thread](https://github.com/apache/beam/pull/17748#discussion_r882989924.) answers this.